apps/libsignpost/sara_u260.c

```diff
@@ -94,26 +113,39 @@ static int sara_u260_del_file(const char* fname) {
 }
 
 static int sara_u260_write_to_file(const char* fname, uint8_t* buf, size_t len) {
+    
+    int ret = at_send(SARA_CONSOLE, "AT+DWNFILE=\"");
+    if (ret < 0) return SARA_U260_ERROR;
+
+    ret = at_send(SARA_CONSOLE, fname);
+    if (ret < 0) return SARA_U260_ERROR;
+
+    ret = at_send(SARA_CONSOLE, "\",");
+    if (ret < 0) return SARA_U260_ERROR;
+
+    char c[15];
+    int clen = snprintf(c,15,"%d",len);
```

Comment:
nit b/c brad has me thinking about proper use of string functions, the format specifier for a size_t is `%zu` not `%d`